packages/client-core/src/admin/components/Setting/Authentication.tsx

```diff
@@ -71,7 +86,7 @@ const Account = (props: Props) => {
     if (user?.id?.value != null && authSettingState.updateNeeded.value) {
       AuthSettingService.fetchAuthSetting()
     }
-  }, [authState.user?.id?.value])
+  }, [authState])
```

Comment:
This won't work unfortunately. Hookstate state objects will change on every component render, so you have to use the actual values for deps here.

Reply:
Eventually (hopefully soon), we'll be able to pass the state the way you did; this is dependent on new feature we are sponsoring in Hookstate.

Reply:
will revert this change and copy the same for other files as well